### PR TITLE
Add Importer support

### PR DESCRIPTION
**Pull request by darkdh**
Auditors: @bridiver, @bbondy, @bsclifton

### CODE REVIEW

File: script/lib/importer.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    'libbookmarks_browser.a',
+    'libcomponent_metrics_proto.a',
+    'libfavicon_base.a',
+    'libgoogle_core_browser.a',
+    'libhistory_core_browser.a',
+    'libsearch_engines.a',
+    'libquery_parser.a',
+    'liburl_formatter.a',
+  ],
+  'linux': [
```
Comment (bridiver) [inline script/lib/importer.py:16]: have you checked these on linux and windows or are these a guess based on the mac libs?

Comment (bsclifton) [reply script/lib/importer.py:16]: @bridiver I know he has tested w/ Windows for sure and I believe Linux was tested w/ Ubuntu.  Is that right, @darkdh?

Comment (darkdh) [reply script/lib/importer.py:16]: I've tested on linux, windows and mac before submitting PR.

Comment (bridiver) [reply script/lib/importer.py:16]: great, thanks!

**Comment by bridiver** [comment]
I'll merge this after I finish updating to 53.0.2785.116

**Comment by bridiver** [comment]
++ just waiting for a small update to https://github.com/brave/libchromiumcontent/pull/4 and I'll merge both and get the builds started

**Pull request merged**